bin/builder_package

```diff
@@ -13,7 +13,7 @@ set -e
 # Build image and push to docker hub:
 #> bin/builder_package latest
 
-DOCKER_IMAGE="mysteriumnetwork/mysterium-node-builder"
+DOCKER_IMAGE="anjmao/mysterium-node-builder"
```

Comment:
This is temporary. After I get push access to mysteriumnetwork I revert this change is separate PR.

Reply:
I've added you to the owners in our org